package com.company.lesson_23;

/* Исключения
Есть метод, который выбрасывает два исключения, унаследованные от Exception, и два унаследованных от RuntimeException:
NullPointerException, ArithmeticException, FileNotFoundException, URISyntaxException.

Нужно перехватить NullPointerException и FileNotFoundException, но не перехватывать
ArithmeticException и URISyntaxException.
*/

import java.io.FileNotFoundException;
import java.net.URISyntaxException;

public class Task_2306 {
    public static void main(String[] args){

    }
    public static void throwsException(){

    }
}
